universal-login-sdk/test/api/wallet/DeployedWallet.int.test.ts

```diff
@@ -153,15 +153,15 @@ describe('INT: DeployedWallet', () => {
       const baseGas = 88720;
       const notEnoughGasLimit = 100;
       message = {...message, gasLimit: baseGas + notEnoughGasLimit};
-      await expect(deployedWallet.execute(message)).to.be.eventually.rejectedWith(`Insufficient Gas. gasLimit should be greater than ${GAS_BASE}`);
+      await expect(deployedWallet.execute(message)).to.be.eventually.rejectedWith(`Insufficient Gas. Got safeTxGas 280 but should greater than ${GAS_BASE}`);
     });
 
     it('Throws when the gas limit is above the relayers maxGasLimit', async () => {
       const {sdk: secondSdk} = await loadFixture(basicSDK);
       message.gasLimit = secondSdk.getRelayerConfig().maxGasLimit + 1;
       const secondDeployedWallet = new DeployedWallet(contractAddress, '', privateKey, secondSdk);
       await expect(secondDeployedWallet.execute(message)).to.be.eventually
-        .rejectedWith('Invalid gas limit. 500001 provided, when relayer\'s max gas limit is 500000');
+        .rejectedWith(`Invalid gas limit. ${secondSdk.getRelayerConfig().maxGasLimit + 1} provided, when relayer's max gas limit is ${secondSdk.getRelayerConfig().maxGasLimit}`);
```

Comment:
Use `message.gasLimit` instead of `secondSdk.getRelayerConfig().maxGasLimit + 1`